config/jobs/kubernetes/sig-testing/kubernetes-kind-presubmits.yaml

```diff
@@ -295,13 +291,13 @@ presubmits:
         - bash
         - -c
         # TODO(bentheelder): remove the mkdir after the next image bump
```

Comment:
drop the TODO?

Reply:
done, still need TODO on the non-canary,